++ date
+ echo '===== ジョブ開始: Fri Aug  8 11:05:45 AM JST 2025 ====='
++ pwd
+ echo 'cwd  = /home/Competition2025/P02/P02U006/ColossalAI'
++ hostname
+ echo 'host = osk-gpu54'
+ echo 'JOB  = 335745'
+ echo 'NODES= osk-gpu[54,56,91]'
+ BASE_LOG_DIR=/home/Competition2025/P02/P02U006/ColossalAI/logs
+ LOG_ROOT=/home/Competition2025/P02/P02U006/ColossalAI/logs/335745
+ export LOG_ROOT
+ mkdir -p /home/Competition2025/P02/P02U006/ColossalAI/logs/335745 /home/Competition2025/P02/P02U006/ColossalAI/logs/335745/tb
+ source /home/Competition2025/P02/P02U006/miniconda3/etc/profile.d/conda.sh
++ export CONDA_EXE=/home/Competition2025/P02/P02U006/miniconda3/bin/conda
++ CONDA_EXE=/home/Competition2025/P02/P02U006/miniconda3/bin/conda
++ export _CE_M=
++ _CE_M=
++ export _CE_CONDA=
++ _CE_CONDA=
++ export CONDA_PYTHON_EXE=/home/Competition2025/P02/P02U006/miniconda3/bin/python
++ CONDA_PYTHON_EXE=/home/Competition2025/P02/P02U006/miniconda3/bin/python
++ '[' -z x ']'
+ conda activate deepseeksft310
+ local cmd=activate
+ case "$cmd" in
+ __conda_activate activate deepseeksft310
+ '[' -n '' ']'
+ local ask_conda
++ PS1=
++ __conda_exe shell.posix activate deepseeksft310
++ '[' -n '' ']'
++ /home/Competition2025/P02/P02U006/miniconda3/bin/conda shell.posix activate deepseeksft310
Traceback (most recent call last):
  File "/home/Competition2025/P02/P02U006/miniconda3/bin/conda", line 13, in <module>
    sys.exit(main())
             ~~~~^^
  File "/home/Competition2025/P02/P02U006/miniconda3/lib/python3.13/site-packages/conda/cli/main.py", line 105, in main
    return conda_exception_handler(main, *args, **kwargs)
  File "/home/Competition2025/P02/P02U006/miniconda3/lib/python3.13/site-packages/conda/exception_handler.py", line 410, in conda_exception_handler
  File "/home/Competition2025/P02/P02U006/miniconda3/lib/python3.13/site-packages/conda/exception_handler.py", line 31, in __call__
  File "/home/Competition2025/P02/P02U006/miniconda3/lib/python3.13/site-packages/conda/exception_handler.py", line 62, in handle_exception
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1022, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1118, in get_code
  File "<frozen importlib._bootstrap_external>", line 1218, in get_data
MemoryError
+ ask_conda=
+ return
